src/frontend/js/translations/fr_FR.po

```diff
@@ -0,0 +1,165 @@
+#. defaultMessage is:
+#. Filter courses
+#: i18n/js/components/SearchFiltersPane/SearchFiltersPane.json
+msgid "Filter courses"
+msgstr ""
+
+#. [settings.filters.new.new] - Possible balue for the "New" filter for courses
+#. defaultMessage is:
+#. First session
+#: i18n/js/settings.json
+msgid "First session"
+msgstr ""
+
+#. [settings.filters.language.fr] - Language
+#. defaultMessage is:
+#. French
+#: i18n/js/settings.json
+msgid "French"
+msgstr ""
+
+#. [components.SearchFilterValueParent.ariaHideChildren] - Accessibility message for the button to hide children of the current filter
+#. defaultMessage is:
+#. Hide child filters
+#: i18n/js/components/SearchFilterValueParent/SearchFilterValueParent.json
+msgid "Hide child filters"
+msgstr ""
+
+#. [settings.filters.language.title] - Title for the "Language" section of course filters (eg. FR / EN etc.)
+#. defaultMessage is:
+#. Language
+#: i18n/js/settings.json
+msgid "Language"
+msgstr ""
+
+#. [components.CourseGlimpse.logoAltText] - Alternate text for the course logo in a course glimpse.
+#. defaultMessage is:
+#. Logo for {courseTitle}
+#: i18n/js/components/CourseGlimpse/CourseGlimpse.json
+msgid "Logo for {courseTitle}"
+msgstr "Logo du cours {courseTitle}"
+
+#. [settings.filters.new.title] - Title for the "New" section of course filters
+#. defaultMessage is:
+#. New courses
+#: i18n/js/settings.json
+msgid "New courses"
+msgstr ""
+
+#. [settings.filters.availability.values.open] - Possible value for the "Availability" filter for courses
+#. defaultMessage is:
+#. Open for enrollment
+#: i18n/js/settings.json
+msgid "Open for enrollment"
+msgstr ""
+
+#. [common.organizationsHumanName] - Title/name to use when we display a list of organizations
+#. defaultMessage is:
+#. Organizations
+#: i18n/js/utils/commonMessages.json
+msgid "Organizations"
+msgstr "Établissements"
+
+#. [components.SearchSuggestField.searchFieldPlaceholder] - Placeholder text displayed in the search field when it is empty.
+#. defaultMessage is:
+#. Search for courses, organizations, categories
+#: i18n/js/components/SearchSuggestField/SearchSuggestField.json
+msgid "Search for courses, organizations, categories"
+msgstr "Recherche des cours, des organisations, des catégories"
+
+#. [components.SearchSuggestField.searchFieldDefaultSearch] - Default query in the main search field. Lets users run a full text search
+#.       with whatever they have typed in.
+#. defaultMessage is:
+#. Search for {query} in courses...
+#: i18n/js/components/SearchSuggestField/SearchSuggestField.json
+msgid "Search for {query} in courses..."
+msgstr "Recherche de {query} dans les cours..."
+
+#. [components.SearchFilterValueParent.ariaShowChildren] - Accessibility message for the button to show children of the current filter
+#. defaultMessage is:
+#. Show child filters
+#: i18n/js/components/SearchFilterValueParent/SearchFilterValueParent.json
+msgid "Show child filters"
+msgstr ""
```

Comment:
missing translations? we can do it later but just making sure it's not a mess-up

Reply:
it's not translated in crowdin.